app/background.js

```diff
@@ -1,3 +1,4 @@
+const electron = require('electron')
 const { dialog, app, globalShortcut } = require('electron')
```

Comment:
I actually tried that, but check this out from [Electron's Screen documentation](https://github.com/electron/electron/blob/master/docs/api/screen.md):

`You cannot require or use this module until the ready event of the app module is emitted.`

So I get one of these when I try that:

![image](https://cloud.githubusercontent.com/assets/11944012/21939470/e4154620-d98d-11e6-9320-c0a0b3016587.png)